btap/src/preprocessing.py

```diff
@@ -142,19 +146,19 @@ def read_hour_energy(path_elec,path_gas,floor_sq):
     Used to read the weather epw file from minio
 
     Args:
-        path: file path where weather.csv file is to be read from in minio
+        path_elec: file path where the electric hourly energy consumed file is to be read from in minio. This is a mandatory parameter and in the case where only one hourly energy output file is provided, the path to this file should be indicated here. 
+        path_gas: This would be path to the gas output file. This is optional, if there is no gas output file to the loaded, then a value of path_gas ='' should be used
         floor_sq: the square foot of the building
     Returns:
        energy_hour_melt: Dataframe containing the clean and transposed hourly energy file.
     """
     
     s3 = acm.establish_s3_connection(acm.settings.MINIO_URL, acm.settings.MINIO_ACCESS_KEY, acm.settings.MINIO_SECRET_KEY)
+    logger.info("%s read_hour_energy s3 connection %s", s3)
```

Comment:
Two placeholders specified, but only one argument.

```suggestion
    logger.info("read_hour_energy s3 connection %s", s3)
```